pages/common/lb.md

```diff
@@ -0,0 +1,28 @@
+# lb
+
+> A blog management tool.
+> More information: <https://github.com/LukeSmithxyz/lb>.
+
+- Make a new draft:
+
+`./lb new`
+
+- Edit a draft:
+
+`./lb edit`
+
+- Delete a draft:
+
+`./lb trash`
+
+- Publish a draft:
```

Comment:
Where does that publish? Is there a list of blog sites this supports?

Reply:
I don't really get your question. It adds the article to the RSS feed, `index.html`, "blogfile" (`2021.html`) and standalone file (`blog/article-name.html`)

Reply:
```suggestion
- Publish a draft to blog directory:
```

Maybe this is clearer?

Reply:
Ah, I didn't think the blog would be self-hosted. The suggestion above looks good (if it's correct), but it should be
```suggestion
- Publish a draft to the blog directory:
```

Reply:
If we put it here, it should also be in `new`'s description. So I would prefer some kind of mentioning it in the general description, like `A tool for managing a blog contained in the current directory.`

Reply:
Sounds good